server/core/src/rodsAgent.cpp

```diff
@@ -194,7 +194,7 @@ irodsAgentSignalExit( int ) {
     // Because the agent factory does not call any of the exec-family functions,
     // the POSIX standard recommends using _exit() instead of exit() to keep the
     // child from corrupting the parent's memory.
-    _exit(1);
+    exit(1);
```

Comment:
Perhaps this should change based on whether ASan is enabled. Or maybe this form of `exit` is correct.

Reply:
Changed to `_exit` because `std::exit` is not async-signal-safe.